packages/releaser/tests/FileManipulator/ChangelogFileManipulator/Source/CHANGELOG-after.md

```diff
@@ -0,0 +1,7 @@
+## v1.0.0 - %d-%d-%d
```

Comment:
is that "%d-%d-%d" correct?

Reply:
Yes, it's a special format PHPUnit uses: https://phpunit.de/manual/6.5/en/appendixes.assertions.html#appendixes.assertions.assertStringMatchesFormat

This is just `number-number-number`